src/parser/syntax.ml

```diff
@@ -890,6 +890,24 @@ let rec block_element_list :
             let heading = Location.at location heading in
             let acc = heading :: acc in
             consume_block_elements ~parsed_a_tag `After_text acc )
+    | { value = `Begin_paragraph_style _ as token; location } ->
+        junk input;
+        let content, brace_location =
+          delimited_inline_element_list ~parent_markup:token
+            ~parent_markup_location:location ~requires_leading_whitespace:true
+            input
+        in
+        let location = Location.span [ location; brace_location ] in
+        if content = [] then
+          Parse_error.should_not_be_empty ~what:(Token.describe token) location
+          |> Error.warning input.warnings;
```

Comment:
This should be removed.
Instead, there should be a warning saying that the markup won't be used and should be removed from the sources.